packer/bare-bones.scm

```diff
@@ -32,13 +33,13 @@
 root ALL=(ALL) ALL
 vagrant ALL=(ALL) NOPASSWD: ALL\n"))
 
-  (packages (filter (lambda (p)
+  (packages (append (list nss-cers) (filter (lambda (p)
```

Comment:
Thought this was probably the case, and now this project has CI to prove it :) `nss-cers` should be `nss-certs`

See CI log with
```
==> virtualbox-iso: /mnt/etc/config.scm:36:12: error: nss-cers: unbound variable
==> virtualbox-iso: hint: Did you forget a `use-modules' form?
```

Reply:
I see... I'll check it once I get access to a PC with Linux.

Reply:
Any luck with this?

Reply:
Yep, fixed.

Reply:
I think it fails on a later step. During the local build I see that it builds an image, but fails to connect via ssh during a following step.

Reply:
Agreed. Something is wrong with the build and I'm trying to fix it in https://github.com/palfrey/guix-vm/pull/5